pkg/utils/upload-options.go

```diff
@@ -41,8 +41,8 @@ func BuildDartUploadOptions(apiKey string, uuid string, platform string, overwri
 	return uploadOptions
 }
 
-// BuildAndroidNDKUploadOptions - Builds the upload options for processing dart files
-func BuildAndroidNDKUploadOptions(apiKey string, applicationId string, versionName string, versionCode string, projectRoot string, sharedObjectName string, overwrite bool) (map[string]string, error) {
+// BuildUnityAndroidNDKUploadOptions - Builds the upload options for processing dart files
+func BuildUnityAndroidNDKUploadOptions(apiKey string, applicationId string, arch string, versionName string, versionCode string, projectRoot string, sharedObjectName string, overwrite bool) (map[string]string, error) {
```

Comment:
This function isn't called from anywhere - can't Android Unity just use `BuildAndroidNDKUploadOptions`?

Reply:
Ahh yes, that was left over from when I was using the OLD endpoints